src/assets.rs

```diff
@@ -401,11 +401,21 @@ fn asset_from_cache<T: serde::de::DeserializeOwned>(
 
 #[cfg(target_os = "macos")]
 fn macos_dark_mode_active() -> bool {
-    let mut defaults_cmd = std::process::Command::new("defaults");
-    defaults_cmd.args(&["read", "-globalDomain", "AppleInterfaceStyle"]);
-    match defaults_cmd.output() {
-        Ok(output) => output.stdout == b"Dark\n",
-        Err(_) => true,
+    const PREFERNECES_FILE: &str = "Library/Preferences/.GlobalPreferences.plist";
+    const STYLE_KEY: &str = "AppleInterfaceStyle";
+
+    let preferences_file = dirs_next::home_dir()
+        .map(|home| home.join(PREFERNECES_FILE))
+        .expect("Could not get home directory");
```

Comment:
I think this could be

```suggestion
    let preferences_file = dirs_next::home_dir()
        .expect("Could not get home directory")
        .join(PREFERNECES_FILE);
```

We could also think about silently ignoring this error (returning a default value). Not sure what a valid scenario for this is, but maybe we shouldn't crash with a panic in this case.

Reply:
I can also do that but code elsewhere panics in this case so I figured it wasn't something `bat` wanted to handle.

Reply:
Ok. Thank you. Let's keep the `.expect(…)`.